CI-CD/AzureDevOps/azure-pipelines-renovate.yml

```diff
@@ -1,25 +0,0 @@
-# add github_com_token & renovate_token as environment variables
-# renovate_token should be an ADO token with admin access to the repos that you would like merge requests opened for
-# github_com token is used for https://docs.renovatebot.com/self-hosting/#githubcom-token-for-release-notes
-
-trigger:
-- master
-
-pool:
-  vmImage: ubuntu-latest
-
-steps:
-- task: WhiteSource@21
-  inputs:
-    cwd: '$(System.DefaultWorkingDirectory)'
-    projectName: '$(Build.Repository.Name)_$(Build.SourceBranchName)'
-  displayName: 'WS Unified Agent Scan'
-
-- script: |
-    docker run --rm -e GITHUB_COM_TOKEN=$(GITHUB_COM_TOKEN) -e RENOVATE_TOKEN=$(RENOVATE_TOKEN) -e RENOVATE_CONFIG -e RENOVATE_ENDPOINT -e RENOVATE_PLATFORM -e RENOVATE_REPOSITORIES renovate/renovate
-  env:
-    RENOVATE_ENDPOINT: https://dev.azure.com/<your-organization>/
-    RENOVATE_PLATFORM: azure
-    RENOVATE_REPOSITORIES: $(System.TeamProject)/$(Build.Repository.Name)
-    RENOVATE_CONFIG: '{"onboardingConfig":{"extends":["github>whitesource/merge-confidence:beta","config:base"]}}'
-  displayName: 'Renovate'
```

Comment:
Why do you want to remove this file?

Reply:
Because it already exists here https://github.com/whitesource-ft/ws-examples/blob/main/Renovate/azure-pipelines.yml